internal/cli/subscribers.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"github.com/windmilleng/tilt/internal/engine"
+	"github.com/windmilleng/tilt/internal/hud"
+	"github.com/windmilleng/tilt/internal/hud/server"
+	"github.com/windmilleng/tilt/internal/sail/client"
+	"github.com/windmilleng/tilt/internal/store"
+)
+
+func ProvideSubscribers(
```

Comment:
out of curiosity, why did this move? did you intend to delete the original one in engine, or did you intend to keep two copies of this function?

Reply:
also, what happened to TiltVersionChecker? did that get dropped on the floor?